omf-python/src/omf1/converter.rs

```diff
@@ -0,0 +1,13 @@
+use omf::omf1::detect_open;
+use pyo3::exceptions::PyIOError;
+use pyo3::prelude::*;
+use std::path::Path;
+
+#[pyfunction]
+pub fn detect_omf1(path: String) -> PyResult<bool> {
```

Comment:
Maybe is_omf1() is clearer that it returns a boolean by the name?

Reply:
cool, I'll fix this up